### PR TITLE
create profile UI

### PR DESCRIPTION
**Pull request by LeDinhHoangAnh**
[Background]

create profile UI
[Side-effect]

None
[Evidence]

evidence

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -128,6 +128,10 @@ dependencies {
     kapt deps.dagger.androidProcessor
     implementation deps.dagger.android
     implementation deps.dagger.androidSupport
+
+    implementation 'com.google.dagger:dagger:2.28.3'
+    kapt 'com.google.dagger:dagger-compiler:2.28.3'
+
```
Comment (NguyenKhacPhuc) [inline app/build.gradle]: remove this

File: app/src/main/java/com/example/android/model/MostPlayedSong.kt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package com.example.android.model
+
+data class MostPlayedSong (var nameMostPlayedSong: String, var artistMostPlayedSong: String, var imageMostPlayedSong: Int)
```
Comment (NguyenKhacPhuc) [inline app/src/main/java/com/example/android/model/MostPlayedSong.kt]: why using var here?

File: gradlew [hunk 3]
```diff
@@ -1,78 +1,129 @@
-#!/usr/bin/env sh
```
Comment (NguyenKhacPhuc) [inline gradlew:1]: revert this  file

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```
Comment (NguyenKhacPhuc) [inline gradle/wrapper/gradle-wrapper.properties]: revert this

File: app/src/main/java/com/example/android/ui/activity/SplashActivity.kt [hunk 5]
```diff
@@ -13,5 +13,8 @@ class SplashActivity(override val toolBar: Toolbar? = null) : BaseActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         DataBindingUtil.setContentView<ViewDataBinding>(this, R.layout.splash_activity)
+
```
Comment (NguyenKhacPhuc) [inline app/src/main/java/com/example/android/ui/activity/SplashActivity.kt:16]: revert this changes

File: app/build.gradle (hunk 1 above)
Comment (LeDinhHoangAnh) [reply app/build.gradle]: removed

File: app/src/main/java/com/example/android/model/MostPlayedSong.kt (hunk 2 above)
Comment (LeDinhHoangAnh) [reply app/src/main/java/com/example/android/model/MostPlayedSong.kt]: using val
